pyproject.toml

```diff
@@ -69,7 +70,6 @@ MarkupSafe = "*"
 Mercurial = "*"
 mrcfile = "*"
 nodeenv = "*"
-nose = "*"
```

Comment:
This and following dependency removals seem wrong?

Reply:
Yep, didn't mean to add that, but that was the only way to have poetry finish resolving.